docs/topics/wasm/wasm-troubleshooting.md

```diff
@@ -0,0 +1,48 @@
+[//]: # (title: Troubleshooting)
+
+Most of the browsers support WebAssembly. For recent browser versions, WebAssembly works by default. However,
+for older browser versions, you still need to update the browser settings.
+
+To run a Kotlin/Wasm project, check if you need to update the settings of the target environment:
+
+<tabs>
+<tab title="Chrome">
+
+* For version 119 or later:
+  
+  It works by default.
+
+* For older versions:
+
+    1. Go to `chrome://flags/#enable-webassembly-garbage-collection` in your browser.
+    2. Enable **WebAssembly Garbage Collection**.
+    3. Relaunch your browser.
+
+</tab>
+<tab title="Firefox">
+
+* For version 120 or later:
+  
+  It works by default.
+
+* For version 119:
+
+  1. Go to `about:config` in your browser.
+  2. Enable `javascript.options.wasm_gc` option.
+  3. Relaunch your browser.
+
+</tab>
+<tab title="Edge">
```

Comment:
How about renaming it to "Chromium-based" and mentioning inside some specific browsers (Edge, Brave, Opera, Samsung Internet)?

Reply:
Alternatively, this tab could be merged into the Chrome tab with a note that it's the same for other Chromium based browsers

Reply:
I renamed it. It is a good idea to merge them, but I noticed the process for updating the settings for older versions is not the same. We would need to keep them in separate tabs (each one with its explanation) unless the 3 steps for Chrome also work for Chromium-based. Do they?

Reply:
> Do they?
It looks like they don't